### ISSUE TITLE
useFunctions TypeError

### ISSUE THREAD

**Issue by dannybarinshtein** [FirebaseExtended/reactfire#197]
Ran into an issue where the useFunctions hook returns the following error:

**TypeError: Cannot read property 'then' of undefined**

The issue is located in **sdk.js** line 23 switch statement where it is missing the functions condition.

```
switch (sdk) {
            case SDK.FIRESTORE:
                sdkPromise = import('firebase/firestore');
                break;
            case SDK.DATABASE:
                sdkPromise = import('firebase/database');
                break;
            case SDK.AUTH:
                sdkPromise = import('firebase/auth');
                break;
            case SDK.STORAGE:
                sdkPromise = import('firebase/storage');
                break;
            case SDK.PERFORMANCE:
                sdkPromise = import('firebase/performance');
                break;
        }
```

**EDIT**

Found related tickets claiming its fixed:
 
https://github.com/FirebaseExtended/reactfire/pull/191
https://github.com/FirebaseExtended/reactfire/issues/189

I am on version "^2.0.0-canary.1fce6b9" - should I be using a different version?

**Comment by jhuleatt** [comment]
Huh, looks like our CI build hasn't released the latest code to canary. Thanks for reporting, @dannybarinshtein !

FYI @jamesdaniels

**Comment by jhuleatt** [comment]
Sorry for the delay - functions and the other packages should be included in today's build: `2.0.0-canary.f3e731d`

**Comment by dannybarinshtein** [comment]
Awesome thanks Jeff - appreciate the update. I actually tried to contribute the fix but when I forked the repo it seemed to be the old one aha

Cheers
